src/runtime/audience-action-local-flows-test.js

```diff
@@ -795,15 +806,23 @@ describes.realWin('AudienceActionLocalFlow', (env) => {
         expect(form).to.not.be.null;
         expect(form.innerHTML).contains('newsletter_code_snippet');
         form.dispatchEvent(new SubmitEvent('submit'));
-        await tick();
+        await tick(3);
 
-        expect(env.win.fetch).to.be.calledTwice;
-        expect(env.win.fetch).to.be.calledWith(
-          'https://news.google.com/swg/_/api/v1/publication/pub1/completeaudienceaction?sut=abc&configurationId=newsletter_config_id&audienceActionType=TYPE_NEWSLETTER_SIGNUP'
+        const updatedWrapper = env.win.document.querySelector(
+          '.audience-action-local-wrapper'
         );
+        expect(updatedWrapper).to.be.null;
         expect(eventManager.logSwgEvent).to.be.calledWith(
           AnalyticsEvent.ACTION_BYOP_NEWSLETTER_OPT_IN_SUBMIT
         );
+        expect(env.win.fetch).to.be.calledTwice;
```

Comment:
This would benefit from a comment explaining why fetch is called twice.

Reply:
Done